releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,581 @@
+
+    [User document](/sql-plan-management.md#baseline-capturing), [#32466](https://github.com/pingcap/tidb/issues/32466)
+
+- Support TiKV quota limiter (experimental)
+
+    If your machine deployed with TiKV has limited resources and the foreground is burdened by an excessively large amount of requests, background CPU resources are occupied by the foreground, causing TiKV performance unstable. In TiDB v6.0, you can use the quota-related configuration items to limit the resources used by the foreground, including CPU and read/write bandwidth. This greatly improves stability of clusters under long-term heavy workloads.
+
+    [User document](/tikv-configuration-file.md#quota), [#12264](https://github.com/tikv/tikv/pull/12264)
+
+- Support the zstd compression algorithm in TiFlash
+
+    TiFlash introduces two parameters, `profiles.default.dt_compression_method` and `profiles.default.dt_compression_level`, which allow users to select the optimal compression algorithm based on performance and capacity balance.
+
+    [User document](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file)
+
+- Enable all I/O checks (Checksum) by default
+
+    This feature was introduced in v5.4.0 as experimental. It enhances data accuracy and security without imposing an obvious impact on users’ businesses.
+
+    Warning: Newer version of data format cannot be downgraded in place to versions earlier than v5.4.0. During such a downgrade, you need to delete TiFlash replicas and replicate data after the downgrade. Alternatively, you can perform a downgrade by referring to [dttool migrate](/tiflash/tiflash-command-line-flags.md#dttool-migrate).
+
+    [User document](/tiflash/use-tiflash.md#use-data-validation)
+
+- Improve thread utilization
+
+    TiFlash introduces asynchronous GRPC and Min-TSO scheduling mechanisms. Such mechanisms ensure more efficient use of threads and avoid system crashes caused by excessive threads.
+
+    [User document](/tiflash/monitor-tiflash.md#coprocessor)
+
+### Data Migration
+
+#### TiDB Data Migration (DM)
+
+- Add WebUI (experimental)
+
+    On the WebUI, you can easily manage a large number of migration tasks. The WebUI supports:
+
+    - Dashboard
+    - Migration task management
+    - Upstream configuration
+    - Replication status query
+    - Master and Worker management
+
+    WebUI is still experimental and is still under development. Therefore, it is recommended only for trial. A known issue is that problems might occur if you use WebUI and dmctl to operate the same task. This issue will be resolved in later versions.
+
+    [User document](/dm/dm-webui-guide.md)
+
+- Add an error handling mechanism
+
+    More commands are introduced to address problems that interrupt a migration task. For example:
+
+     - In case of a schema error, you can update the schema file by using the `--from-source/--from-target` parameter of the `binlog-schema update` command, instead of editing the schema file separately.
```

Comment:
```suggestion
    - In case of a schema error, you can update the schema file by using the `--from-source/--from-target` parameter of the `binlog-schema update` command, instead of editing the schema file separately.
```